https://issues.redhat.com/browse/ACM-21686 (#8217)
open-cluster-management/rhacm-docs: clusters/release_notes/mce_known_issues.adoc

Learn about known issues and limitations during {mce-short} installation.

[#unclaimed-hive-clusters]
=== Unclaimed Hive clusters might be recreated during ClusterPool custom resource definition changes
//2.9:ACM-21686

> During an {mce-short} upgrade or modification to the `ClusterPool` custom resource definition, Hive might delete and recreate all unclaimed clusters. This is expected behavior and ensures that unclaimed clusters reflect the latest configuration defined in the updated `ClusterPool` resource specification.

Author: ```suggestion
//2.9:ACM-21686

During an {mce-short} upgrade or modification to the `ClusterPool` custom resource definition, Hive might delete and recreate all unclaimed clusters. This is expected behavior and ensures that unclaimed clusters reflect the latest configuration defined in the updated `ClusterPool` resource specification.
```

> During an {mce-short} upgrade or modification to the `ClusterPool` custom resource definition, Hive might delete and recreate all unclaimed clusters. This is expected behavior and ensures that unclaimed clusters reflect the latest configuration defined in the updated `ClusterPool` resource specification.

Reviewer: ```suggestion
During a {mce-short} upgrade or modification to the `ClusterPool` custom resource definition, Hive might delete and recreate all unclaimed clusters. This is expected behavior and ensures that unclaimed clusters reflect the latest configuration defined in the updated `ClusterPool` resource specification.
```

> link:https://issues.redhat.com/browse/ACM-21686[ACM-21686]

Author: ```suggestion

link:https://issues.redhat.com/browse/ACM-21686[ACM-21686]
```

Learn about known issues and limitations during {mce-short} installation.

> === Unclaimed Hive clusters might be recreated during ClusterPool CRD resource changes

Author: ```suggestion
[#unclaimed-hive-clusters]
=== Unclaimed Hive clusters might be recreated during ClusterPool custom resource definition changes
```

=== Unclaimed Hive clusters might be recreated during ClusterPool custom resource definition changes
//2.9:ACM-21686

> //still editing

Author: ```suggestion

```

> During an upgrade or modification to the ClusterPool custom resource definition (CRD)—such as those that may occur implicitly through an MCE upgrade—Hive may delete and recreate all unclaimed clusters. This is expected behavior and ensures that unclaimed clusters reflect the latest configuration defined in the updated ClusterPool specification.

Author: ```suggestion
//still editing
During an {mce-short} upgrade or modification to the `ClusterPool` custom resource definition, Hive might delete and recreate all unclaimed clusters. This is expected behavior and ensures that unclaimed clusters reflect the latest configuration defined in the updated `ClusterPool` resource specification.
```

During an {mce-short} upgrade or modification to the `ClusterPool` custom resource definition, Hive might delete and recreate all unclaimed clusters. This is expected behavior and ensures that unclaimed clusters reflect the latest configuration defined in the updated `ClusterPool` resource specification.

link:https://issues.redhat.com/browse/ACM-21686[ACM-21686]

> 

Author: ```suggestion
link:https://issues.redhat.com/browse/ACM-21686[ACM-21686]

```

Learn about known issues and limitations during {mce-short} installation.

[#unclaimed-hive-clusters]
=== Unclaimed Hive clusters might be recreated during ClusterPool custom resource definition changes
//2.9:ACM-21686
//still editing
During an {mce-short} upgrade or modification to the `ClusterPool` custom resource definition, Hive might delete and recreate all unclaimed clusters. This is expected behavior and ensures that unclaimed clusters reflect the latest configuration defined in the updated `ClusterPool` resource specification.

> link:https://issues.redhat.com/browse/ACM-21686[ACM-21686]

Author: ```suggestion
link:https://issues.redhat.com/browse/ACM-21686[ACM-21686]
```
